bin/main

```diff
@@ -56,40 +61,20 @@ def player_turn(arr, name, symbol, remaining_slots, player_arr)
   end
 end
 
-until tie
-  design_board(slots)
-
-  remaining_slots = slots.select { |item| item.is_a? Integer }
-
-  player_turn(slots, player1.name, 'X', remaining_slots, player1_arr)
-  sleep 1
-
-  if check_win(player1_arr, winning_numbers)
-    puts "#{player1.name.capitalize} wins"
-    break
-  end
-
-  if slots.none? { |item| item.is_a? Integer }
-    puts "It's a Tie!!"
-    break
-  end
-
-  remaining_slots = slots.select { |item| item.is_a? Integer }
-
-  design_board(slots)
-
-  player_turn(slots, player2.name, 'O', remaining_slots, player2_arr)
-
-  remaining_slots = slots.select { |item| item.is_a? Integer }
-  sleep 1
-
-  if check_win(player2_arr, winning_numbers)
```

Comment:
- The game should display the board at winning condition before it returns a win message. 
![Screenshot from 2021-03-27 02-54-04](https://user-images.githubusercontent.com/64894927/112696890-b7d00000-8e97-11eb-9c91-e52236119c16.png)